packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -88,6 +92,30 @@ describe("Garbage Collection Tests", () => {
         clock.restore();
     });
 
+    describe("Session expiry is called", () => {
+        beforeEach(() => {
+            closeCalled = false;
+        });
+        it("Session expiry is called with existing timeout", async () => {
+            const metadata: IContainerRuntimeMetadata = 
+                { summaryFormatVersion: 1, message: undefined, sessionExpiryTimeoutMs: 1 }
+            createGarbageCollector(undefined, undefined, metadata);
+            clock.tick(1);
+            assert(closeCalled, "Close should have been called.");
+        });
+
+        it("Session expiry is called", async () => {
+            const settings = { "Fluid.GarbageCollection.RunSessionExpiry": "true" };
```

Comment:
nit: Move the initialization of settings into the "beforeEach" block and the reset to old config in "afterEach". I believe all the tests in this describe block would want this setting enabled. So, additional tests won't have to worry about it and the test becomes cleaner too.